src/app/app.module.ts

```diff
@@ -92,6 +97,9 @@ const routes: Routes = [
         component: HeaderToolbarComponent
       },
       { path: 'domainregistration', component: DomainRegisterComponent},
+      { path: 'identities', component: ProfilesComponent},
+      { path: 'dev', component: DevComponent},
```

Comment:
Not sure if we should have this in a production build. It's harmless of course, but feels a bit off. Perhaps we could move it to a separate app, like with have with rmm6 or e2e currently: something that we can build ourselves, but not something we ship to the customers?

Reply:
Maintain another app just for this? i rather have it available to check easily... and also simple to link from if necessary.